src/omnibus/omnibus.py

```
@@ -256,7 +259,14 @@ def recv_message(self, timeout: int | None = None) -> Message | None:
             )
         return None
 
-    def recv(self, timeout: int | None = None) -> Any | None:
+    def new_recv(self, timeout: int | None = None) -> RLCSv3Message:
```

Ruff (0.14.5)

Unused method argument: `timeout`

(ARG002)